src/index.css

```diff
@@ -1,13 +1,24 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu",
+    "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  padding-left: 56px;
+  transition: padding-left 0.2s ease-out;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
+  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New", monospace;
+}
+
+@media (min-width: 325px) {
+  .with-panel {
+    padding-left: 56px;
+  }
+}
+@media (min-width: 1200px) {
+  .with-panel {
+    padding-left: 280px;
```

Comment:
The effect is nice! :) and it's good that for smaller screens it doesn't change the body width. But you added `padding-left: 56px` both for "body" and small .with-panel

Reply:
56px is for workspaces sidebar, I guess it should persist even on smaller screens.

Reply:
what i mean is
```
@media (min-width: 325px) {
  .with-panel {
    padding-left: 56px;
  }
}
```
is a duplicate of 
```
body {
  padding-left: 56px;
}
``` ;)